### ISSUE TITLE
Classic UI 1.8.1 and new slider: major problem

### ISSUE THREAD

**Issue by lolodomo** [openhab/openhab#3985]
Using the new slider provided with Classic UI in openHAB 1.8.1, I succeeded several times to enter in a loop with infinite update of the item (always with the same value). This is clearly the classic UI that update the item because if I close the WEB browser, the updates stop.
Unfortunately, it is not clear for me what is the exact scene to reproduce this issue.

**Comment by BClark09** [comment]
I'm getting the same thing, as soon as I change the brightness sliders on the classic UI, my lights go crazy, switching between different states, undefined really fast. In a single second my openHAB server recieves this many updates:

```
[ip address] -  -  [07/Feb/2016:01:59:21 +0000] "GET /CMD?BR_L_Bright=86&__async=true&__source=wa0000 HTTP/1.1" 200 13 
[ip address] -  -  [07/Feb/2016:01:59:21 +0000] "GET /CMD?BR_L_Bright=86&__async=true&__source=wa0000 HTTP/1.1" 200 13 
[ip address] -  -  [07/Feb/2016:01:59:21 +0000] "GET /CMD?BR_L_Bright=86&__async=true&__source=wa0000 HTTP/1.1" 200 13 
[ip address] -  -  [07/Feb/2016:01:59:21 +0000] "GET /CMD?BR_L_Bright=86&__async=true&__source=wa0000 HTTP/1.1" 200 13 
[ip address] -  -  [07/Feb/2016:01:59:21 +0000] "GET /CMD?BR_L_Bright=86&__async=true&__source=wa0000 HTTP/1.1" 200 13 
[ip address] -  -  [07/Feb/2016:01:59:21 +0000] "GET /CMD?BR_L_Bright=86&__async=true&__source=wa0000 HTTP/1.1" 200 13 
[ip address] -  -  [07/Feb/2016:01:59:21 +0000] "GET /CMD?BR_L_Bright=86&__async=true&__source=wa0000 HTTP/1.1" 200 13 
[ip address] -  -  [07/Feb/2016:01:59:21 +0000] "GET /openhab.app?sitemap=default&w=0000&poll=true&__async=true&__source=wa0000 HTTP/1.1" 200 4145 
[ip address] -  -  [07/Feb/2016:01:59:21 +0000] "GET /CMD?BR_L_Bright=86&__async=true&__source=wa0000 HTTP/1.1" 200 13 
[ip address] -  -  [07/Feb/2016:01:59:21 +0000] "GET /CMD?BR_L_Bright=86&__async=true&__source=wa0000 HTTP/1.1" 200 13 
[ip address] -  -  [07/Feb/2016:01:59:21 +0000] "GET /CMD?BR_L_Bright=86&__async=true&__source=wa0000 HTTP/1.1" 200 13 
```

**Comment by teichsta** [comment]
Thanks for letting us know! I'll try to find a solution asap …

**Comment by BClark09** [comment]
Hi @teichsta thanks for looking into it, any progress on this?

Would you recommend going back to 1.8.0 in the mean-time?

**Comment by teichsta** [comment]
> Hi @teichsta thanks for looking into it, any progress on this?

yes and no … i decided to revert the slider change for 1.8 for various reasons (see PR).

**Comment by BClark09** [comment]
Ah that's helpful, thanks for the information, and the work that you do.

**Comment by teichsta** [comment]
problem has been fixed by reverting the slider (see #4051)
